acra-dialog/src/main/java/org/acra/annotation/AcraDialog.java

```diff
@@ -52,7 +46,7 @@
      * @return Class for the CrashReportDialog used when prompting the user for crash details.
```

Comment:
Updated the dialog class doc to mention `CrashReportDialogHelper` since otherwise the developer might not know about it.

```suggestion
     * @return The activity to use to prompt the user for crash details. Inside the activity, use {@link org.acra.dialog.CrashReportDialogHelper} to integrate it ACRA.
```